package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "vivaro-website",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "react": "^18",
```

Comment:
it's better to use exact versions. Rather than ^

Reply:
done